vowpalwabbit/learner.cc

```diff
@@ -45,12 +45,13 @@ void learn_multi_ex(multi_ex& ec_seq, vw& all)
   as_multiline(all.l)->finish_example(all, ec_seq);
 }
 
+template <bool finish_example=true>
```

Comment:
Can be removed - the example cannot be finished early because that erases `ec->end_pass` information.